src/foundry_dev_tools/cli/git_credential_foundry.py

```diff
@@ -0,0 +1,66 @@
+"""A git credential helper.
```

Comment:
Can you add docs for this feature? How exactly is the python code called from git?